contracts/protocol/bases/ProtocolBase.sol

```diff
@@ -535,6 +533,23 @@ abstract contract ProtocolBase is PausableBase, ReentrancyGuardBase {
 
         // Offer must not already be voided
         require(!offer.voided, OFFER_HAS_BEEN_VOIDED);
+    }
+
+    /**
+     * @notice Gets offer and seller from protocol storage
+     *
+     * Reverts if:
+     * - Offer does not exist
+     * - Offer already voided
+     * - Seller assistant is not the caller
+     *
+     *  @param _offerId - the id of the offer to check
+     */
```

Comment:
Just for consistency, maybe add return parameters to natspec.

And now looking at the places where it's used. apparently, it doesn't even have to return the seller.
I guess you can remove it from return. 
Maybe it then needs only a better name (I know I suggested this one :grin: ), so it doesn't sound like it's returning the seller. Sthi like, `getValidOfferWithSellerCheck`? I'm so bad at this, if you think of anything better, just use it instead.

Reply:
done